src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-snapshot-list/rbd-snapshot-list.component.html

```diff
@@ -33,6 +33,12 @@
             <span *ngIf="selection.first()?.is_protected"><i class="fa fa-fw fa-unlock"></i><span i18n>Unprotect</span></span>
           </a>
         </li>
+        <li role="menuitem"
+            [ngClass]="{'disabled': !selection.hasSingleSelection || selection.first().executing || !selection.first().is_protected}">
```

Comment:
Starting w/ Mimic, you do **not** need to use protected snapshots to clone an image if clone v2 is enabled. It would be based on the ```rbd_default_clone_format``` configuration setting:

```1```: require a protected snapshot
```2```: do not require a protected snapshot
```auto```: check the OSD maps' ```min_compat_client``` / ```require_min_compat_client``` values and if the minimum client is at least Mimic, do not require a protected snapshot.